enhancements/installer/aws-load-balancer-type.md

```diff
@@ -0,0 +1,345 @@
+
+```yaml
+ oc edit ingress.config/cluster -o yaml
+ # Please edit the object below. Lines beginning with a '#' will be ignored,
+ # and an empty file will abort the edit. If an error occurs while saving this file will be
+ # reopened with the relevant failures.
+ #
+ apiVersion: config.openshift.io/v1
+ kind: Ingress
+ metadata:
+   creationTimestamp: "2022-08-30T01:03:17Z"
+   generation: 11
+   name: cluster
+   resourceVersion: "60888"
+   uid: de89cc95-449e-4ae5-a837-f4171fe61e36
+ spec:
+   domain: apps.testnlbapi.devcluster.openshift.com
+   loadBalancer:
+     platform:
+       aws:
+         type: NLB
+       type: AWS
+ status:
+   componentRoutes:
+```
+
+- The cluster-ingress-operator will check the cluster ingress config object for the lbType and reconcile to create a default ingress controller.
+- If the user-specified ingress controller CR does not specify LB type then the LB Type defined in the ingress config will be used for reconcilation
+  by the ingress operator to create the LB.
+
+#### Destroying cluster
+
+- None
+
+
+### Risks and Mitigations
+
+- Users need to put either "NLB" or "Classic" as input or else the LB creation will fail.
+- To mitigate this we need to validate the input.
+
+### Drawbacks
+
+None
+
+## Design Details
+
+### Test Plan
+
+- Set the `lbType` field in the `install-config.yaml` either to "NLB" or "Classic" and see if the default ingress controller gets created with the desired LB type.
+- Delete the default ingress controller CR and see if the default ingress controller will be created using the LB type mentioned during installation.
+- Unit tests and E2E tests will be written.
+
+### Graduation Criteria
+
+*This enhancement will follow standard graduation criteria.
+
+#### Dev Preview -> Tech Preview
+
+- N/A
+
+#### Tech Preview -> GA
+
+- N/A
+
+#### Removing a deprecated feature
+
+None
+
+### Upgrade / Downgrade Strategy
+
+Upgrade won't have issues as far as the installer and OCP version(as ingress operator has the fix) have the feature.
+
+Downgrade to a version of installer and OCP will create CLB/ELB by default.
+
+### Version Skew Strategy
+
+Not applicable.
+
+### Operational Aspects of API Extensions
+
+None
+
+#### Failure Modes
+
+Users need to put either "NLB" or "Classic" for lbType or omit the field or else the install-config will fail validation
+
+#### Support Procedures
+
+If the goal is to determine how the ingress operator is determining the default LB type, you could also suggest checking the 
+cluster version or ingress operator version to make sure it is new enough to have the enhancement, using `oc version` or 
+`oc get clusteroperators/ingress -o yaml`, and checking the LB type in the cluster ingress config, using `oc get ingress.config.openshift.io/cluster -o yaml`. 
+You could also tell the user how to check the LB type that the operator specified on the service, by checking the appropriate 
+annotations in `oc -n openshift-ingress get svc/router-default -o yaml`.
```

Comment:
The wording here is confusing.  It looks like you just copied and pasted my comment: https://github.com/openshift/enhancements/pull/1148#discussion_r979528038